### DISCUSSION TITLE
Error: Request failed with status code 404

### DISCUSSION THREAD

**Discussion by Alpha1022** [gitalk/gitalk#445] [Q&A]
#### Gitalk 版本
1.7.2

#### 报错信息？
Error: Request failed with status code 404

#### 浏览器版本？
Firefox 86.0

#### 复现步骤？
已有评论可正常显示，但试图使用 GitHub 登录时将出现上述报错。

**Comment by vecpeng** [comment]
同样的问题

**Reply by marcushaghes** [reply]
Are you willing to know who your spouse really is, if your spouse is cheating just contact zeusmasterhack he is good at hacking into cell phones, changing school grades and many more this great hacker has also worked for me and I got results of spouse what's-app messages, call logs, text messages,  Facebook, emails. deleted text messages and many more this hacker is very fast cheap and affordable he has never disappointed me for once contact him if you have any form of hacking problem am sure he will help you THANK YOU.
contact: zeusmasterhack@gmail.com

**Comment by cnrkuo** [comment]
確保 OAuth app id 正確
